packages/devtools_app/lib/src/inspector/inspector_breadcrumbs.dart

```diff
@@ -1,3 +1,9 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
please change this to 2021, as that's the year this file was added

Reply:
changed locally